### DISCUSSION TITLE
How to create a nav menu item

### DISCUSSION THREAD

**Discussion by mre-aydogdu** [onweru/newsroom#47] [Q&A]
Hi, I just can't figure out how to add a nav menu item. I managed to do that by adding .md files at /content outside the /public, but the issue is I can't create pages inside. Simply put, I want the layout of the e.g. "Tags" menu item to stay the same but change what's inside, how can I do that? Thanks.

**Comment by Lsm-CVX** [comment]
Jsut add a file called menu.yaml into the data/ folder (you can also use toml if you prefer. 

`- item: Home
  url: /
- item: About
  url: /about/
- item: Contact
  url: /contact/
`
